.travis.yml

```diff
@@ -7,7 +7,11 @@ sudo: false
 script:
   - cargo build --verbose
   - cargo test --verbose
-  - [ "$TRAVIS_RUST_VERSION" = "1.0" ] || cargo test --verbose --features no_std
+  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
+       cargo test --verbose --features no_std
```

Comment:
Can you be sure to test `no_std` on non-1.0 as well? It should be tested on beta at least and likely on stable as well.